packages/abi-gen/package.json

```diff
@@ -11,38 +11,37 @@
         "lint": "tslint --format stylish --project . && yarn lint-contracts",
         "lint-contracts": "solhint -c ../../contracts/.solhint.json test-cli/contracts/*.sol",
         "fix": "tslint --fix --format stylish --project . && yarn lint-contracts",
-        "clean": "shx rm -rf lib test-cli/**/generated-wrappers",
-        "build": "tsc -b",
-        "test_cli:clean": "yarn clean:sol && rm -rf test-cli/**/generated-wrappers && rm -rf test-cli/typescript/lib",
-        "test_cli:prebuild": "yarn compile:sol && yarn generate_contract_wrappers",
-        "test_cli:build": "tsc --project test-cli/tsconfig.json",
-        "test_cli": "yarn test_cli:run_mocha && yarn diff_contract_wrappers",
-        "build:ci": "yarn build && yarn test_cli:prebuild && yarn test_cli:build",
-        "test": "yarn run_mocha",
-        "run_mocha": "mocha --require source-map-support/register --require make-promises-safe lib/test/*_test.js --timeout 100000 --bail --exit",
-        "test_cli:run_mocha": "mocha --require source-map-support/register --require make-promises-safe test-cli/typescript/lib/**/*_test.js --timeout 100000 --bail --exit",
+        "clean": "shx rm -rf lib test-cli/**/output",
+        "build": "tsc -b && yarn generate_contract_wrappers && yarn test_cli:build",
+        "build:ci": "yarn build",
+        "test": "yarn run_mocha && yarn test_cli",
         "test:circleci": "yarn test:coverage && yarn test_cli",
+        "run_mocha": "mocha --require source-map-support/register --require make-promises-safe lib/test/*_test.js --timeout 100000 --bail --exit",
         "test:coverage": "nyc npm run test --all && yarn coverage:report:lcov",
         "coverage:report:lcov": "nyc report --reporter=text-lcov > coverage/lcov.info",
+        "test_cli": "yarn test_cli:run_mocha && yarn diff_contract_wrappers",
+        "test_cli:clean": "yarn clean:sol && rm -rf test-cli/**/output && rm -rf test-cli/typescript/lib",
```

Comment:
Remove `yarn clean:sol` from here.  The way it is right now, running this will result in a git status of missing files, which will be very confusing to someone new to this code.

Reply:
good catch